Assignment/training3/src/components/Modal/Modal.js

```diff
@@ -0,0 +1,60 @@
+import React, {useState, useEffect} from 'react';
+import PropTypes from 'prop-types';
+import './Modal.css';
+
+
+Modal.PropTypes = {
+    hide: PropTypes.bool,
+}
+
+Modal.default = {
+    hide: true,
+}
+
+
+function Modal(props) {
+    const {hide, setHide} = props;
+
+    const handleModal = (e) => {
+
+        setHide(false);
+    }
+
+    const handleKeyDown = (e) => {
+        console.log(e.key);
+    }
+
+
+    useEffect(() =>{
+
+        // componentDidMount
+        window.addEventListener('keydown', handleKeyDown);
```

Comment:
@lehuy1012  I want add listener when hide change to false, and remove when hide change to true, not mount, unmount

Reply:
fixed